schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id integer,
```

Comment:
- [ ] [Optional] You can add 
- `int NOT NULL IDENTITY(1,1)`
- `PRIMARY KEY (id)`

Reply:
Great Idea, thanks for this advice 😄